PluginGenerator/Maven/MavenArtifactHandler.cs

```diff
@@ -109,13 +109,13 @@ private string GetArtifactUrl(IMavenCoordinate coordinate, string extension)
         private string GetArtifactFolderPath(IMavenCoordinate coordinate)
         {
             string path = string.Format(System.Globalization.CultureInfo.InvariantCulture,
-                "{0}/{1}/{2}",
-                coordinate.GroupId.Replace(".", "/"),
+                "{0}\\{1}\\{2}",
+                coordinate.GroupId.Replace(".", "\\"),
```

Comment:
Windows tolerates paths containing both "/" and "\" but it's neater to use one or the other.